index.d.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Supported license types
+ */
+type LicenseType =
+  | 'agpl'
+  | 'apache'
+  | 'artistic'
+  | 'bsd-3-clause'
+  | 'bsd'
+  | 'cc0'
+  | 'eclipse'
+  | 'gpl-v2'
+  | 'gpl-v3'
+  | 'isc'
+  | 'lgpl-v2.1'
+  | 'lgpl-v3'
+  | 'mit'
+  | 'mozilla'
+  | 'no-license'
+  | 'unlicense'
+  | 'wtfpl'
+
+/**
+ * Create license options
+ * @property year years
+ * @property author author
+ * @property project project name
+ */
+type Options = {
+  year: number
+  author: string
+  project: string
+}
+
+/**
+ * Create a license
+ * @param where Directory for creating licenses
+ * @param type Type of license
+ * @param opts Some options
+ */
+export default function createLicense(
+  where: string,
+  type: LicenseType,
+  opts: Options,
+): void
+
+/**
+ * Get a list of all supported licenses
+ */
+export const licenses: [
```

Comment:
Should this be =?

Reply:
No, `.d.ts` is a type declaration file, so only the type is declared and no specific assignment is made.
Reference: https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html